Cargo.lock

```diff
@@ -723,9 +723,9 @@ checksum = "55b672471b4e9f9e95499ea597ff64941a309b2cdbffcc46f2cc5e2d971fd335"
 
 [[package]]
 name = "compiler_builtins"
-version = "0.1.105"
+version = "0.1.108"
```

Comment:
Just to note, this may conflict with the pin from https://github.com/rust-lang/rust/pull/120454#issuecomment-1931944439 if that lands.

Reply:
I've r-'d that one. I'll rebase once this lands so we don't conflict